src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -213,27 +213,17 @@ class JooqTouchstoneRepository(
                                    activityType: ActivityType,
                                    supportLevel: GAVISupportLevel): Int
     {
-        try
-        {
-            val support = supportLevel.toString().toLowerCase()
-            val activity = activityType.toString().toLowerCase().replace('_', '-')
-            val record = this.dsl.newRecord(COVERAGE_SET).apply {
-                this.touchstone = touchstoneVersionId
-                this.name = "$vaccine: $vaccine, $support, $activity"
-                this.vaccine = vaccine
-                this.gaviSupportLevel = support
-                this.activityType = activity
-            }
-            record.store()
-            return record.id
-        }
-        catch(e: DataAccessException)
```

Comment:
Now that we have a generic way of returning foreign key errors, don't need this custom logic